web/packages/teleterm/src/services/appUpdater/autoUpdatesStatus.ts

```diff
@@ -53,8 +53,22 @@ export async function resolveAutoUpdatesStatus(sources: {
     };
   }
 
-  const { reachableClusters, unreachableClusters } =
-    await sources.getClusterVersions();
+  const clusterVersions = await sources.getClusterVersions();
+  return findManagingCluster({
+    clusterVersions,
+    managingClusterUri: sources.managingClusterUri,
+  });
+}
+
+/**
+ * Attempts to find a cluster managing updates, based on a user-selected cluster
+ * and all available clusters.
+ */
+function findManagingCluster(sources: {
```

Comment:
Idk how accurate the name and the comment are since it technically it just finds a version rather than a cluster?

Reply:
Changed.